app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
```

Comment:
Check order of imports + is a good practice to separate different kinds of imports with blank lines